### ISSUE TITLE
bingo-elastic reaction euclid test fails during CI

### ISSUE THREAD

**Issue by khyurri** [epam/Indigo#321]
Test `api/plugins/bingo-elastic/java/src/test/java/com/epam/indigo/predicate/ReactionsTest.java` failed, because result differs from expected. 

I changed reaction test file, CI is working right now, but issue should be investigated

Changed code:
```java
IndigoRecordReaction targetReaction = reactions.get(3);
```
From 
```java
IndigoRecordReaction targetReaction = reactions.get(2);
```

**Comment by MysterionRise** [comment]
@khyurri isn't it fixed now?

**Comment by khyurri** [comment]
@MysterionRise 
Nope, fixed only test, but reason of previously failed test  should be investigated.
